package.json

```diff
@@ -67,9 +69,11 @@
     "typescript": "~2.3.4"
   },
   "peerDependencies": {
-    "lodash": "4.x"
+    "lodash": "4.x",
+    "tslib": "^1.7.1"
   },
   "dependencies": {
-    "tslib": "^1.6.1"
+    "lodash": "^4.17.4",
```

Comment:
That is intentional. It should use your applications version of lodash. This avoids having lodash version mismatches.